CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+# Contributing
+
+Pull requests are welcome!
+
+## Getting started
+
+First [fork this repo](https://help.github.com/articles/fork-a-repo/), and
+then clone it & set the upstream repo:
+
+```bash
+git clone git@github.com:your-github-username/halyard.git
+
+   # in my case (username is lwander), the above command looks like:
+   # git clone git@github.com:lwander/halyard.git
+
+cd halyard
+
+git remote add-url upstream git@github.com:spinnaker/halyard.git
+
+  # the above command allows you to push patches to the upstream repo for 
+  # approval
+```
+
+## Configuring IntelliJ
```

Comment:
I read this more as
1. How do I get the code?
2. How to read open it in my editor?
3. How do I run it?
4. Where do I make changes?

which is what I'd like to get out of a contributing guide.